react-native-ssi/example/src/App.tsx

```diff
@@ -28,20 +30,22 @@ export function App() {
               style={styles.button}
               onPress={() => {
                 generateDidKey('RSA').then((result) => {
-                  addLogLine(result.didKey);
                   setDidKey(result.didKey);
+                  addLogLine(didKey);
+
                   setPrivateJwk(result.privateJwk);
+                  addLogLine(privateJwk);
                 });
               }}
             >
               <Text style={styles.buttonText}>Generate DID</Text>
             </TouchableOpacity>
             <TouchableOpacity
-              disabled={didKey == ''}
+              disabled={didKey === ''}
```

Comment:
i never use the `==`. maybe a good rule to add to linter

Reply:
because it's easy to miss the difference between `==` and `===`  but also because there is usually a cleaner and shorter way to coerce. for example
 - `disabled={!didKey}` (preferable) or
 -  `disabled={Boolean(myStr)}`